### PR TITLE
Example documents

### PR DESCRIPTION
**Pull request by LucvandenBrand**
Summarized all our document explanations posted on Nestor in the readme, so we can simply post the readme and the files next year if so desired.
Added some example documents based on their final grade.

### CODE REVIEW

File: example-documents/README.md [hunk 1]
```diff
@@ -1,3 +1,50 @@
-this folder contains example documents 
-from past teams that we consider are of
-high quality. 
\ No newline at end of file
+This folder contains example documents from past teams that we consider are of high quality. 
+Use them as a rough guideline, not as a template.
+
+As a general note: try to keep your documents **consise**. That is: short but descriptive.
```
Comment (mircealungu) [inline example-documents/README.md]: concise

File: example-documents/README.md [hunk 2]
```diff
@@ -1,3 +1,50 @@
...
+The sad truth of the world is that people do not like to read large documents, and if your document is not being read you are wasting valuable development time ;).
+
+## Requirements Document
+The requirements document should be self contained and should  also make sense to somebody who is not part of the team. Thus it should have a high-level introduction that explains the importance and the scope of the project. But the most important role of this document, is that it will be the evolving shared understanding between you and the customer. Thus, the more details it contains the more can you be sure that you are on the same page. And this is why we ask you to update it after every meeting with the customer. 
+
```
Comment (mircealungu) [inline example-documents/README.md:9]: Imagine that you are writing the document for a team new member who will join the project halfway through. Or even for a team that will have to continue with the project once you are finished. Will they understand what needs to be done?

File: example-documents/README.md [hunk 3]
```diff
@@ -1,3 +1,50 @@
...
+- **Critical Functional Requirements**:
+Requirements that your client would be very angry about if they are missing in the final product.
+- **Important Functional Requirements**:
+Requirements that would disappoint your client if they were missing in the final product.
+- **Useful Functional Requirements**:
+Requirements that your client would be happy about if they were implemented, but it's OK if they are not.
+- **Non-Functional Requirements**:
+All requirements that describe a certain _quality_ of the product, such as "Intuitive design" or "Android support". Be sure to properly understand the distinction between functional and non-functional (_a large part of students initially lose points due to this mistake!_).
+- **Won't Do**:
+Features you will not implement. Noting what you _will not_ do is just as important as noting what you _will_ do.
```
Comment (mircealungu) [inline example-documents/README.md:26]: why? :)

Comment (LucvandenBrand) [reply example-documents/README.md:26]: A team might decide during a meeting that they will not support iOS _ever_ (not really a useful decision in my opinion, but they might do it), but their client might forget this choice (or due to a miscommunication never actually agreed to this) and could demand this feature during the final set of iterations. Angry clients and sad teams ensue if it is not documented.
Should I add that?

File: example-documents/README.md (hunk 2 above)
Comment (LucvandenBrand) [reply example-documents/README.md:9]: If they follow the guidelines (and TA feedback), I hope so.
We could ask students to read each-others documents more next year, as I do not know if we have any past data indicating how readable the highly graded documents _actually_ are for those who are not involved already. 

We could also switch the second grader more often, as it woud force us as well to more properly review the descriptiveness of these documents.

**Pull request merged**